docs/_pages/advanced_numerical.rst

```diff
@@ -0,0 +1,145 @@
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install theano
+
+Keras
+----------
+
+Keras is another Python library for machine learning using neural networks. It is capable of interacting with other machine learning libraries, including TensorFlow and Theano. Keras runs on CPUs and GPUs and is designed for fast implementation of neural networks.
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install keras
+
+LmFit
+-----
+
+LmFit is a great tool for non-linear curve fitting. It uses SciPy under-the-hood, but offers a better interface. Specifically, it offers more control when estimating model parameters. If you're going to be fitting complex data in Python, we suggest LmFit.
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install lmfit
+
+emcee
+-----
+
+emcee is a Python library built for efficient Bayesian computations. It implements the Affine Invariant Markov chain Monte Carlo (MCMC) Ensemble sampler to reconstruct posterior probability distributions for model parameters. If you're looking to implement a Bayesian fit in your analysis this libary will be a useful tool. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install emcee
+
+PyMC3
+----
+
+PyMC3 is another useful tool for implementing Bayesian inference in your analyses. PyMC3 is a versatile probabilistic programming framework that allows users to define probabilistic models directly in Python. Under the hood it uses a variety of clever trickes to make computations faster. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install pymc3
+
+Networkx
+--------
+
+Networkx is an excellent library designed to deal with network analyses. It introduces a Graph datastructure that makes encoding networks easy and intuitive. Networkx includes a variety of tools for building and analyzing networks. It has the ability to export Graph objects in many different formats and can interface directly with NumPy and Pandas. If you're looking to deal with networks in your analysis Networkx will be extremely useful. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install networkx
+
+Numba
+-----
+
+Numba is a library designed to help you speed up your Python calculations. It achieves this goal using a just-in-time compiler, which gives Python code speed that is comparable in performance to C. Numba is easy to use. Python functions can be wrapped with a simple decorator that results in increased speed. 
+
+To install this library in your Python environment simply run: 
+::
+
+    pip install scikit-learn numba
+
+Cython
+------
+
+Cython is another library aimed at speeding up Python code. It effectively allows users to write C code using Python syntax. Cython code is compiled and runs at C speed, increasing the performance of your code. 
```

Comment:
Cython is another library aimed at speeding up Python code. Users can write Python code and quickly translate it to a C extension.

Reply:
:+1: